components/user_settings/security/user_access_token_section/user_access_token_section.jsx

```diff
@@ -78,37 +78,38 @@ export default class UserAccessTokenSection extends React.PureComponent {
                 saving: false,
             };
         }
-        return {active: nextProps.active};
+        return { active: nextProps.active };
     }
 
     startCreatingToken = () => {
-        this.setState({tokenCreationState: TOKEN_CREATING});
+        this.setState({ tokenCreationState: TOKEN_CREATING });
     }
 
     stopCreatingToken = () => {
-        this.setState({tokenCreationState: TOKEN_NOT_CREATING, saving: false});
+        this.setState({ tokenCreationState: TOKEN_NOT_CREATING, saving: false });
     }
 
     handleCreateToken = async () => {
         this.handleCancelConfirm();
+        this.newtokendescription = React.createRef();
```

Comment:
Can you rename for consistency with other PRs?
```suggestion
        this.newtokendescriptionRef = React.createRef();
```